docs/content/Contribution guidelines/operations/packaging-and-versioning.md

```diff
@@ -56,8 +56,8 @@ Following are some recent tags and what they correspond to.
 
 ## Core container image
 
-The core container runs the kcp server and the KubeStellar core
-controllers and initializes the kcp workspaces. It is built from some
+The core container runs a space provider server and the KubeStellar core
```

Comment:
The proposed new language here is not accurate for providers other than kcp. For kind there isn't even a "server", and KubeFlex has its own container image.

Reply:
removed 'server' - my mistake

Reply:
The problem here is not just the word "server". KubeFlex has its own machinery, including container image and Pod that runs it.

BTW, this is also entangled with a problem that existed before this PR. The text "The core container runs the kcp server and the KubeStellar core containers" is a bit mixed up.  For example, it is saying "The core container runs ... containers". That is not right.  That text confuses container _image_ with _container_ and suggests (with the singular "the") that there is only one core container. We currently run five core containers, all using one container image: kcp server, init, and three controllers.

Here is a suggested revision that is accurate:

> The core container image is used in containers that run the KubeStellar core containers, initialize the KubeStellar core, and run one of the supported space providers.

Reply:
right - changed